examples/example_notebooks/stable_baselines3_example.ipynb

```diff
@@ -0,0 +1,456 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from stable_baselines3 import DQN\n",
+    "from stable_baselines3.dqn import MlpPolicy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2 Setting the parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the DQN algorithm you have to define a set of values. The policy_kwargs dictionary is a parameter which is directly given to the MlpPolicy. The net_arch key defines the network architecture of the MLP."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "buffer_size = 200000 #number of old obsersation steps saved\n",
+    "learning_starts = 10000 # memory warmup\n",
+    "train_freq = 1 # prediction network gets an update each train_freq's step\n",
+    "batch_size = 25 # mini batch size drawn at each update step\n",
+    "policy_kwargs = {\n",
+    "        'net_arch': [64,64] # hidden layer size of MLP\n",
+    "        }\n",
+    "exploration_fraction = 0.1 # Fraction of training steps the epsilon decays \n",
+    "target_update_interval = 1000 # Target network gets updated each target_update_interval's step\n",
+    "gamma = 0.99\n",
+    "verbose = 1 # verbosity of stable basline's prints"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Additionally, you have to define how long your agent will train. You can just set a concrete number of steps or use knowledge of the environment's temporal resolution to define an in-simulation training time. In this example the agent is trained for five seconds which translates in this environment's case to 500000 steps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tau = 1e-5\n",
+    "simulation_time = 5 # seconds\n",
+    "nb_steps = int(simulation_time // tau)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.3 Starting the training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once you've setup the environment and defined your parameters starting the training is nothing more than a one-liner. For each algorithm all you have to do is call its .learn() function. However, you should note that the execution of the training can take a long time. Currently, Stable Baselines3 does not provide any means of reward observation during training. Therefore, a RewardLogger callback is used for this environment (see code a few cells above)."
```

Comment:
"Currently, Stable Baselines3 does not provide any means of reward observation during training." 
If you mean monitoring the reward visually, this is in no way part of an agent such that this will never find its way into SB3.